app/configurator/components/select-dataset-step.tsx

```diff
@@ -113,16 +113,41 @@ export const SelectDatasetStepContent = () => {
         }}
       >
         <Box sx={{ maxWidth: 900 }}>
-          <Text variant="heading1" sx={{ mb: 4 }}>
-            {dataset ? null : filters.length > 0 ? (
-              filters
-                .filter((f) => f.__typename !== "DataCubeAbout")
-                .map((f) => (f.__typename !== "DataCubeAbout" ? f.label : null))
-                .join(", ")
-            ) : (
-              <Trans id="browse.datasets.all-datasets">All datasets</Trans>
-            )}
-          </Text>
+          {dataset ? null : filters.length > 0 ? (
+            filters
+              .filter((f) => f.__typename !== "DataCubeAbout")
+              .map((f) => (f.__typename !== "DataCubeAbout" ? f.label : null))
```

Comment:
Do we need this condition in .map? Maybe a type guard in filter would work?

Reply:
Ah, it worked finally: c0a7315

Reply:
Yes, I was testing the same approach 😅